curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/659ebe52d74b132a1d75c891.md

```diff
@@ -0,0 +1,388 @@
+---
+id: 659ebe52d74b132a1d75c891
+title: Step 46
+challengeType: 0
+dashedName: step-46
+---
+
+# --description--
+
+If you try to add a new task, edit that task, and then click on the `Add New task` button, you will notice a bug.
+
+The form button will display the incorrect text of `Update Task` instead of `Add Task`. To fix this, you will need to update your `addOrUpdateTask` function to update the `innerText` of the `addOrUpdateTaskBtn` to `Add Task`.
```

Comment:
Let's update this part too so it is more clearer for beginners

```suggestion
The form button will display the incorrect text of `Update Task` instead of `Add Task`. To fix this, you will need to assign the string `Add Task` to `addOrUpdateTaskBtn.innerText` inside your `addOrUpdateTask` function.
```